src/network/network-er-x-sfp.md

```diff
@@ -0,0 +1,332 @@
+set service dhcp-server shared-network-name 100.64.10.0 subnet 100.64.10.0/24 default-router 100.64.10.1
+set service dhcp-server shared-network-name 100.64.10.0 subnet 100.64.10.0/24 dns-server 1.1.1.1
+set service dhcp-server shared-network-name 100.64.10.0 subnet 100.64.10.0/24 domain-name to.mesh
+set service dhcp-server shared-network-name 100.64.10.0 subnet 100.64.10.0/24 lease 600
+set service dhcp-server shared-network-name 100.64.10.0 subnet 100.64.10.0/24 start 100.64.10.127 stop 100.64.10.254
+
+set service dhcp-server shared-network-name 100.64.11.0 authoritative enable
+set service dhcp-server shared-network-name 100.64.11.0 subnet 100.64.11.0/24 default-router 100.64.11.1
+set service dhcp-server shared-network-name 100.64.11.0 subnet 100.64.11.0/24 dns-server 1.1.1.1
+set service dhcp-server shared-network-name 100.64.11.0 subnet 100.64.11.0/24 domain-name to.mesh
+set service dhcp-server shared-network-name 100.64.11.0 subnet 100.64.11.0/24 lease 600
+set service dhcp-server shared-network-name 100.64.11.0 subnet 100.64.11.0/24 start 100.64.11.127 stop 100.64.11.254
+
+set service dhcp-server shared-network-name 100.64.12.0 authoritative enable
+set service dhcp-server shared-network-name 100.64.12.0 subnet 100.64.12.0/24 default-router 100.64.12.1
+set service dhcp-server shared-network-name 100.64.12.0 subnet 100.64.12.0/24 dns-server 1.1.1.1
+set service dhcp-server shared-network-name 100.64.12.0 subnet 100.64.12.0/24 domain-name to.mesh
+set service dhcp-server shared-network-name 100.64.12.0 subnet 100.64.12.0/24 lease 600
+set service dhcp-server shared-network-name 100.64.12.0 subnet 100.64.12.0/24 start 100.64.12.127 stop 100.64.12.254
+
+set service dhcp-server shared-network-name 100.64.13.0 authoritative enable
+set service dhcp-server shared-network-name 100.64.13.0 subnet 100.64.13.0/24 default-router 100.64.13.1
+set service dhcp-server shared-network-name 100.64.13.0 subnet 100.64.13.0/24 dns-server 1.1.1.1
+set service dhcp-server shared-network-name 100.64.13.0 subnet 100.64.13.0/24 domain-name to.mesh
+set service dhcp-server shared-network-name 100.64.13.0 subnet 100.64.13.0/24 lease 600
+set service dhcp-server shared-network-name 100.64.13.0 subnet 100.64.13.0/24 start 100.64.13.127 stop 100.64.13.254
+```
+
+## L2TP Tunnel
+
+L2TP Tunnels is used to connect back to the an [exit node](exit-node.md).  
+
+All commands are prefixed with `set interfaces l2tpv3 l2tpeth0` where `l2tpeth0` is a numbered interface starting from 0.
+
+Between the EXIT NODE configuration and the CLIENT node the following values are filled 
+
+```
+- destination-port <> source-port
+- session-id <> peer-session-id
+- tunnel-id <> peer-tunnel-id
+- remote-ip <> local-ip
+```
+
+`local-ip` MUST be an IP address defined on the router. This means if the device is behind a NAT it will be a local IP not public IP. Also, if using NAT sometimes source-port needs to be added to the port forward.
```

Comment:
```suggestion
`local-ip` must be an IP address defined on the router. This means if the device is behind a NAT, this will be a local IP, and not the public IP. Also, if the device is behind NAT, sometimes the `source-port` needs to be added to the router's port forward settings.
```